common-lib/build.gradle

```diff
@@ -11,7 +11,7 @@ def ideaHome = '../idea-IC/'
 dependencies {
     compile files('lib/GoogleFeedback.jar')
     compile fileTree(dir: ideaHome + '/lib')
-    compile "org.mockito:mockito-all:1.9.5"
+    compile files('third_party/mockito/mockito-all-1.9.5.jar')
```

Comment:
yes, that's already been pushed and merged into this client. I think Github is showing a diff against the state of the code whehn this branch was created.